examples/ATF_script.lua

```diff
@@ -0,0 +1,432 @@
+Test = require('connecttest') 
```

Comment:
@VVeremjova, Can you rename this file, for example 'Common_ATF_script.lua'. Because, now it's not clearly for me what is it.